rsts/user/getting_started/create_first.rst

```diff
@@ -37,89 +34,61 @@ The most basic Flyte primitive is a "task". Flyte Tasks are units of work that c
 
 Start by creating a new file ::
 
-   mkdir -p workflows
-   touch workflows/first.py
 
-This directory has been marked in the `configuration file <https://github.com/lyft/flytesnacks/blob/764b82aca5701137ebc0eda4e818466e5acc9219/sandbox.config#L2>`_ as the location to look for workflows and tasks.  Begin by importing some of the libraries that we'll need for this example.
+   touch cookbook/recipes/core/first.py
+
 
 .. code-block:: python
 
-  from __future__ import absolute_import
-  from __future__ import division
-  from __future__ import print_function
-  
-  import urllib.request as _request
+  from urllib import request
   
   import cv2
-  from flytekit.common import utils
-  from flytekit.sdk.tasks import python_task, outputs, inputs
-  from flytekit.sdk.types import Types
-  from flytekit.sdk.workflow import workflow_class, Output, Input
+  import flytekit
+  from flytekit import task, workflow
+  from flytekit.types.file import FlyteFile
   
-From there, we can begin to write our first task.  It should look something like this. 
+From there, we can begin to write our first task.  It should look something like this:
 
 .. code-block:: python
 
-  @inputs(image_location=Types.String)
-  @outputs(parsed_image=Types.Blob)
-  @python_task
-  def edge_detection_canny(wf_params, image_location, parsed_image):
-      with utils.AutoDeletingTempDir('test') as tmpdir:
-          plane_fname = '{}/plane.jpg'.format(tmpdir.name)
-          with _request.urlopen(image_location) as d, open(plane_fname, 'wb') as opfile:
-              data = d.read()
-              opfile.write(data)
-  
-          img = cv2.imread(plane_fname, 0)
-          edges = cv2.Canny(img, 50, 200)  # hysteresis thresholds
-  
-          output_file = '{}/output.jpg'.format(tmpdir.name)
-          cv2.imwrite(output_file, edges)
-  
-          parsed_image.set(output_file)
+  @task
+  def edge_detection_canny(image_location:str) -> FlyteFile:
+      working_dir = flytekit.current_context().working_directory:
+      plane_fname = '{}/plane.jpg'.format(working_dir.name)
+      with request.urlopen(image_location) as d, open(plane_fname, 'wb') as opfile:
+          data = d.read()
+          opfile.write(data)
+
+      img = cv2.imread(plane_fname, 0)
+      edges = cv2.Canny(img, 50, 200)  # hysteresis thresholds
+
+      output_file = '{}/output.jpg'.format(working_dir.name)
+      cv2.imwrite(output_file, edges)
+
+      return FlyteFile["jpg"](path=output_file)
```

Comment:
can we say that you can iterate on this code by running the example

edge_detection_canny(image_location-"")..

Reply:
done.